src/pages/HomePage.vue

```diff
@@ -9,48 +9,59 @@
             exist. Yikes.
           </p>
         </div>
-        <BaseButton buttonType="primary" caption="Book a ticket"/>
-   
+        <BaseButton
+          buttonType="primary"
+          caption="Book a ticket"
+        />
+
       </div>
       <div class="main_image"></div>
     </section>
     <div class="more-info">
       <p>soon <span>in the cinema</span></p>
-      <p class="showItems" @click="changeVisible">see all</p>
+      <p
+        class="showItems"
```

Comment:
I'd suggest to be consistent in how we name our classes and stick to it - you have `kebab-case` everywhelse else so `show-items` would be much more fitting 😄 
```suggestion
        class="show-items"
```

Reply:
Yes, you're right. Sometimes there is a chaos in names of these classes...